Reneiski_Homework1/Zavod/Worker.cs

```diff
@@ -1,27 +1,42 @@
 ﻿namespace WorkerNamespace
 {
-    public class Worker
+    public class Worker : Person
     {
-        private string firstName;
-        private string lastName;
-        private int age;
         private string position;
 
         public Worker(string firstName, string lastName, int age, string position)
+            : base(firstName, lastName, age)
         {
-            this.firstName = firstName;
-            this.lastName = lastName;
-            this.age = age;
             this.position = position;
         }
 
-        public Worker()
+        public Worker() : base(null, null, 0)
```

Comment:
It's better to create constructor without parameters in base class rather then pulling null and 0

Reply:
Fixed